nafas/functions.py

```diff
@@ -294,21 +299,19 @@ def run(program_data):
     :type program_data: dict
     :return: None
     """
-    sound_check()
+    sound_check_flag = sound_check()
     cycle = program_data["cycle"]
     ratio = program_data["ratio"]
     unit = program_data["unit"]
     pre = program_data["pre"]
     print("Preparing ", end="", flush=True)
-    sid = play_sound(get_sound_path(SOUND_MAP['Prepare']))
+    play_sound(get_sound_path(SOUND_MAP['Prepare']), enable=sound_check_flag)
```

Comment:
Why we wrap the `play_sound` function into a `if sound_check_flag`?
I prefer it because that way we do not change the `play_sound` function, which has now a more natural signature.

Reply:
If that's the case, we should use `if sound_check_flag` whenever we want to play sound. It doesn't seem to be efficient.